build-mods-pages.js

```diff
@@ -147,6 +147,20 @@ function filepath(parts) {
 }
 
 for (const mod of mods) {
+    if (mod.versions[0].modJSON.developer == undefined && mod.versions[0].modJSON.developers != undefined) {
```

Comment:
jesus christ just make a variable for the current version

Reply:
made some variables